packages/twenty-front/src/modules/activities/emails/components/Emails.tsx

```diff
@@ -26,22 +30,54 @@ const StyledEmailCount = styled.span`
   color: ${({ theme }) => theme.font.color.light};
 `;
 
-export const Emails = () => (
-  <StyledContainer>
-    <Section>
-      <StyledH1Title
-        title={
-          <>
-            Inbox <StyledEmailCount>{emails.length}</StyledEmailCount>
-          </>
-        }
-        fontColor={H1TitleFontColor.Primary}
-      />
-      <Card>
-        {emails.map((email, index) => (
-          <EmailPreview divider={index < emails.length - 1} email={email} />
-        ))}
-      </Card>
-    </Section>
-  </StyledContainer>
-);
+export const Emails = ({ entity }: { entity: ActivityTargetableEntity }) => {
```

Comment:
nit: I find it confusing to mention ActivityTarget here, we probably want to have a different type even though it represents the same thing 🤔 
cc @charlesBochet